lampod/src/lib.rs

```diff
@@ -106,6 +113,45 @@ impl LampoDaemon {
         &self.conf
     }
 
+    pub fn liquidity(&self) -> Arc<LampoLiquidityManager> {
+        self.liquidity.clone().unwrap()
+    }
+
+    // This should not be initiated when we open our node only when we
+    // provide some args inside cli or in lampo.conf
+    fn init_lampo_liquidity(
+        &self,
+        node_id: PublicKey,
+        addr: SocketAddress,
+        token: Option<String>,
+    ) -> error::Result<LampoLiquidityManager> {
+        log::info!("Starting lampo-liquidity manager as a consumer!");
+        //TODO: Fix this!
+        let liquidity = LampoLiquidity::new(
+            self.offchain_manager().key_manager(),
+            self.channel_manager().manager(),
+            Some(self.onchain_manager()),
+            None,
+            None,
+            None,
+        );
+        let liquidity = LampoLiquidityManager::new(
+            liquidity.into(),
+            Arc::new(self.conf.clone()),
```

Comment:
`Arc::new(self.conf.clone())` does not look right, why do you need to pass a pointer?